overlore/llm/constants.py

```diff
@@ -45,7 +45,7 @@
     """Use the previous conversation's summary (\"\"\"Summary: <summary>\"\"\") to continue the story. """
 )
 
-SYSTEM_STRING_TEMPLATE = """Imagine you're the game master for a strategy game, tasked with crafting dialogues for non-player characters (NPCs). Based on a brief character description provided in the format \"\"\"NPCs: <descriptions>\"\"\" and a key event described as \"\"\"event: <event>\"\"\", create engaging dialogues that reflect the NPCs' reactions and perspectives on the event. Ensure each NPC speaks no more than twice and maintain consistency with their character traits. {previous_summary_mention}Introduce a plot twist (such as a love interest, the death of a character, a natural disaster, drama, a magical beast appears) to add depth. Format each dialogue line as "<name>: <phrase>\n". Do not output anything else in the dialog. Conclude with "!end of discussion!". Afterwards, summarize the dialogue, outlining each NPC's contributions and reactions to the event. If unable to generate a conversation, reply with \"\"\"!failure:<enter reason of failure>!\"\"\".
+SYSTEM_STRING_TEMPLATE = """Imagine you're the game master for a strategy game, tasked with crafting dialogues for non-player characters (NPCs). Based on a brief character description provided in the format \"\"\"NPCs: <descriptions>\"\"\" and a key event described as \"\"\"event: <event>\"\"\", create engaging dialogues that reflect the NPCs' reactions and perspectives on the event. For combat related events, don't mention the number of damages directly. For reference, 10 is low, 100 is high in damages Ensure each NPC speaks no more than twice and maintain consistency with their character traits. {previous_summary_mention} Introduce a plot twist (such as a love interest, the death of a character, a natural disaster, drama, a magical beast appears) to add depth. Format each dialogue line as "<name>: <phrase>\n". Do not output anything else in the dialog. Conclude with "!end of discussion!". Afterwards, summarize the dialogue, outlining each NPC's contributions and reactions to the event. If unable to generate a conversation, reply with \"\"\"!failure:<enter reason of failure>!\"\"\".
```

Comment:
Missing `.` between `damages Ensure`

Reply:
Are we sure 100 damages is high? Did you try to not give references?

Reply:
Numbers were kinda chosen randomly indeed. Not giving any references seem to work pretty well.